tests/cases/fourslash/goToDefinitionMetaProperty.ts

```diff
@@ -0,0 +1,8 @@
+/// <reference path='fourslash.ts' />
+
+////im/*1*/port.met/*2*/a;
+////function t() { n/*3*/ew.t/*4*/arget; }
+
+for (const marker of test.markerNames()) {
+    verify.goToDefinition(marker, []);
+}

```

Comment:
@weswigham added. Is this the right way of doing this test? I copied from goToDefinitionBuiltInTypes/gotoDefinitionBuiltInValues.

Reply:
Yeah - you might want to also have a version where the file has an `export` and so is flagged as a module.

Reply:
I think it's the opposite we need in that case—the `externalModuleIndicator` will be set when an `import.meta` expression exists. I updated it to also have a test when not a module in 4ff9898